src/resolve/connectionResolver.ts

```diff
@@ -31,10 +31,20 @@ export class ConnectionResolver {
   private connection: Connection;
   private registry: RegistryAccess;
 
-  public constructor(connection: Connection, registry = new RegistryAccess()) {
+  // Array of metadata type names to use for listMembers. By default it includes
+  // all types defined in the registry.
+  private mdTypeNames: string[];
+
+  public constructor(connection: Connection, registry = new RegistryAccess(), mdTypes?: string[]) {
```

Comment:
not sure if we export ConnectionResolver or not, but I'd switch to a an Object when param #3 shows up.

Reply:
It's exported so it'd have to be a major bump for that change.